src/tseda/datastore.py

```diff
@@ -71,31 +70,31 @@ class IndividualsTable(Viewer):
         default=100,
         doc="Number of rows per page to display",
     )
-    toggle = param.Integer(
-        default=None, bounds=(0, None), doc="Toggle sample set by index"
+    sample_select = pn.widgets.MultiChoice(
+        name="Select sample sets",
+        description="Select samples based on the sample set ID.",
+        options=[],
+        value=[0, 1],
```

Comment:
should the default be all or none instead of 1 & 2 specifically? And if all is the default, there should maybe be an "unselect all" option?

Reply:
This is a quick fix because structure currently does not allow for nothing being selected. You need at least two sample groups to be selected for it to not throw an error. I added this as an issue in #53